src/pages/DummySearchVolunteerEvents.jsx

```diff
@@ -1,5 +1,79 @@
+import { useState, useEffect } from 'react';
+import { Select, Input } from '@chakra-ui/react';
+import { fetchEvents, fetchJoinedEvents } from '../../utils/fuseUtils';
+import Fuse from 'fuse.js';
+import JoinedDataContainer from '../components/DummySearchVolunteerEvents/JoinedDataContainer';
+
 const DummySearchVolunteerEvents = () => {
-  return <p>Placeholder for the search volunteer events page</p>;
+  const [eventsData, setEventsData] = useState([]);
+  const [joinedData, setJoinedData] = useState([]);
+  const [searchResults, setSearchResults] = useState([]);
+  const [volunteerResults, setVolunteerResults] = useState([]);
+  const [input, setInput] = useState('');
+
+  /* 
+    This useEffect is for fetching all the events and JOINED events/volunteers/events_data data
+  */
+  useEffect(() => {
+    const setData = async () => {
+      await fetchEvents().then(data => setEventsData(data));
+      await fetchJoinedEvents().then(data => {
+        const joinedContainers = data.map(event => {
+          return <JoinedDataContainer data={event} key={event.volunteer_id} />;
+        });
+        setJoinedData(joinedContainers);
+      });
+    };
+    setData();
+  }, []);
+
+  /*
+    This useffect is used for updating the display volunteer data based on user input utilizing fuzzy search
+  */
+  useEffect(() => {
+    const options = {
+      keys: ['props.data.first_name', 'props.data.last_name', 'props.data.email'],
+    };
+    const fuse = new Fuse(searchResults, options);
+    const searchResult = fuse.search(input);
+    const reduceResult = searchResult.map(result => result.item);
+    setVolunteerResults(reduceResult);
+  }, [input]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'searchResults'. Either include it or remove the dependency array.